fiona/drvsupport.py

```diff
@@ -151,14 +155,25 @@
           'PCIDSK': (2, 0, 0),
           'GeoJSONSeq': (2, 4, 0)},
 
-    'a': {'GMT': (2, 0, 0),
-          'GPKG': (1, 11, 0),
+    'a': {'GPKG': (1, 11, 0),
           'GeoJSON': (2, 1, 0),
-          'MapInfo File': (2, 0, 0),
-          'PCIDSK': (2, 0, 0)}    
+          'MapInfo File': (2, 0, 0)}
 }
 
 
+def driver_supports_mode(driver, mode):
+    """ Returns True if driver supports mode, False otherwise"""
+
+    if driver not in supported_drivers:
+        return False
+    if mode not in supported_drivers[driver]:
+        return False
+    if driver in driver_mode_mingdal[mode]:
+        if get_gdal_version_num() < calc_gdal_version_num(*driver_mode_mingdal[mode][driver]):
+            return False
+    return True
```

Comment:
@rbuffat would you be okay with making this a "private" function in 1.8.14, i.e. changing the name to `_driver_supports_mode`?